tests/functional/checkmate/views/ui/api/add_to_allow_list_test.py

```diff
@@ -22,7 +22,7 @@ def test_a_request_with_a_session_cookie_succeeds(self, app, json_body):
                     "rule": "example.com/",
                     "tags": ["manual"],
                 },
-                "id": 1,
+                "id": response.json["data"]["id"],
```

Comment:
Now that we add multiple rules in functional tests we can't rely on `id` here being constant.